app/controllers/legacy_routes/redirect_url.rb

```diff
@@ -10,8 +10,8 @@
 class LegacyRoutes::RedirectUrl
 
   LEGACY_PATHS = {
-    teams: /(groups)$/,
-    groups: /(accounts)$/,
+    teams: /(folders)$/,
```

Comment:
folders is not a legacy path !
in this controller, we handle all legacy routes. so routes that existed before the url simplifying story.
we only need to redirect legacy routes ... e.g. /en/teams/42/groups/23